### PR TITLE
Mtdsim 93 Refactor front end code urls

### PR DESCRIPTION
**Pull request by FearMyCode**

### CODE REVIEW

**Comment by nicodeas** [comment]
can you please create a `.env.example` file with default values ie what they are now

**Comment by nicodeas** [comment]
then people can just copy that into a .env file for local development

**Comment by nicodeas** [comment]
then you can update setup information / steps with your new steps. 😄

**Review by Vobrymn — APPROVED** [review]

File: client/src/stores/simulation.js [hunk 1]
```diff
@@ -22,20 +23,8 @@ export const useSimulationStore = defineStore("simulation", {
...
       };
-      if (this.parameters.startTime) {
-        reqBody.startTime = Number(this.parameters.startTime);
-      }
-      if (this.parameters.finishTime) {
-        reqBody.finishTime = Number(this.parameters.finishTime);
-      }
-      if (this.parameters.mtdInterval) {
-        reqBody.mtdInterval = Number(this.parameters.mtdInterval);
-      }
```
Comment (Vobrymn) [inline client/src/stores/simulation.js:33]: these params seem to have been accidentally deleted

**Review by Vobrymn — CHANGES_REQUESTED** [review]

File: client/src/stores/simulation.js (hunk 1 above)
Comment (Vobrymn) [reply client/src/stores/simulation.js:33]: probably by me whoops

**Review by Vobrymn — APPROVED** [review]

**Pull request merged**